src/Storages/StorageKeeperMap.cpp

```diff
@@ -790,6 +782,7 @@ void StorageKeeperMap::drop()
     ops.emplace_back(zkutil::makeRemoveRequest(zk_tables_path, -1));
     ops.emplace_back(zkutil::makeCreateRequest(zk_dropped_path, "", zkutil::CreateMode::Persistent));
     ops.emplace_back(zkutil::makeCreateRequest(zk_dropped_lock_path, "", zkutil::CreateMode::Ephemeral));
+    ops.emplace_back(zkutil::makeSetRequest(zk_dropped_lock_version_path, toString(getStorageID().uuid), -1));
```

Comment:
maybe `toString(getStorageID().uuid)` -> `toString(storage_id.hasUUID() ? storage_id.uuid : storage_id.getFullTableName())` ?

Reply:
KeeperMap cannot be used without UUID.
But what I can reuse is `table_unique_id` which I set in storage constructor